packages/db/prisma/schema.prisma

```diff
@@ -13,10 +13,10 @@ datasource db {
 }
 
 model User {
-  id         Int         @id @default(autoincrement())
-  clerkId    String      @unique
-  absences   Absence[]
-  pushTokens PushToken[]
+  id           Int           @id @default(autoincrement())
+  clerkId      String        @unique
+  participants Participant[]
```

Comment:
Each user only needs to be linked to one participant

Reply:
wait, i'll edit absence and participant model to use userId

Reply:
ok! the ticket said participants should have a many-to-many relationship with user field which is why i did many-to-many relation.

Reply:
mb thanks for fixing